src/modules/blocks/attendees/template.js

```diff
@@ -82,27 +88,78 @@ const RenderSubtitle = () => (
 	</div>
 );
 
-const Attendees = ( props ) => {
-
-	const { isSelected, title } = props;
+const renderUI = ( props ) => {
+	const { isSelected, title, displayTitle, displaySubtitle } = props;
 	const blockTitle = ! ( isSelected || title )
 		? renderPlaceholder()
 		: [ renderLabelInput( props ) ];
 
 	return (
 		<div className="tribe-editor__block tribe-editor__event-attendees">
-			{ blockTitle }
-			{ <RenderSubtitle /> }
+			{ displayTitle ? blockTitle : '' }
+			{ displaySubtitle ? <RenderSubtitle /> : '' }
 			{ <RenderGravatars /> }
 		</div>
 	);
 };
 
+const renderControls = ( {
+	isSelected,
+	displayTitle,
+	displaySubtitle,
+	onSetDisplayTitleChange,
+	onSetDisplaySubtitleChange,
+} ) => (
+	isSelected && (
+		<InspectorControls key="inspector">
+			<PanelBody title={ __( 'Attendees Settings', 'events-gutenberg' ) }>
+				<ToggleControl
+					label={ __( 'Display Title', 'events-gutenberg' ) }
+					checked={ displayTitle }
+					onChange={ onSetDisplayTitleChange }
+				/>
+				<ToggleControl
+					label={ __( 'Display Subtitle', 'events-gutenberg' ) }
+					checked={ displaySubtitle }
+					onChange={ onSetDisplaySubtitleChange }
+				/>
+			</PanelBody>
+		</InspectorControls>
+	)
+);
+
+class Attendees extends Component {
+
+	componentDidMount() {
+		const { onKeyDown, onClick } = this.props;
+		document.addEventListener( 'keydown', onKeyDown );
+		document.addEventListener( 'click', onClick );
+	}
+
+	componentWillUnmount() {
+		const { onKeyDown, onClick } = this.props;
+		document.removeEventListener( 'keydown', onKeyDown );
+		document.removeEventListener( 'click', onClick );
+	}
+
+	render() {
+		return [
```

Comment:
Nothing to change here as it works at it's but I think it can make things a bit more _Reactish_, as the values from `renderUI` and `renderControls` are components you can refactor to change the name and the values here into something like.

```js
return [
	<UI {...props} />,
	<Controls {...props} />,
];
```

Just need to make sure `UI` and  `Controls` are capitalized also removing the `render` as they are components the `render on calling them is implicit.